### PR TITLE
included the announcement for April 21

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**
included the announcement for April 21

### CODE REVIEW

File: announcements.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 | Date  | Announcement |
 | ------------- |:-------------:|
-| 05 April 2023 | Some of the Windows users would have been unable to connect to GCC 2.0 or SGTS services due to an issue with Cloudflare WARP. For more information, see [SEED service status](seed-status?id=previous-incidents). | 
+| 20 April 2023 | We will be performing a scheduled maintenance on our SEED servers on 21 April 2023. For more information, see [SEED service status](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/seed-status?id=scheduled-maintenance). |
+| 05 April 2023 | Some of the Windows users would have been unable to connect to GCC 2.0 or SGTS services due to an issue with Cloudflare WARP. For more information, see [SEED service status](seed-status?id=previous-incidents). |
```
Comment (eu-tee) [inline announcements.md:6]: @SenthilKumarSivakami this one should be removed?

Comment (SenthilKumarSivakami) [reply announcements.md:6]: @eu-tee , Do you mean to remove the record of the previous incident from the announcement page?

Comment (eu-tee) [reply announcements.md:6]: sorry, i saw that u removed it at the top, then i realised this is the announcement page. Keep it then.

Comment (SenthilKumarSivakami) [reply announcements.md:6]: cool, thanks!

**Review by samloh84 — APPROVED** [review]
LGTM!

**Pull request merged**